packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -188,6 +190,22 @@ export default class Meeting extends StatelessSparkPlugin {
     });
   }
 
+  getAudioMuted() {
+    return this.audio.getMuted();
+  }
+
+  getAudioSelf() {
+    return this.audio.getSelf();
+  }
+
+  getVideoMuted() {
```

Comment:
Should this be `isVideoMuted()`? Does it return a boolean or something else? Same goes for audio above and I'm not sure what self is

Reply:
sure, pick one 'is' or 'get'

Reply:
`is` please

Reply:
done